ecms_base/features/custom/ecms_publications/tests/src/Functional/EcmsPublicationsInstallTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\ecms_publications\Functional;
+
+// Require the all profiles abstract class since autoloading doesn't work.
+require_once dirname(__FILE__) . '/../../../../../../../tests/src/Functional/AllProfileInstallationTestsAbstract.php';
+
+use Drupal\Tests\ecms_profile\Functional\AllProfileInstallationTestsAbstract;
+
+/**
+ * Class InstallationTest.
```

Comment:
Nitpick, but this should be `Class EcmsPublicationsInstallTest.`.

Reply:
Got it!